plugins/inputs/disk/disk_test.go

```diff
@@ -74,13 +75,13 @@ func TestDiskUsage(t *testing.T) {
 	assert.Equal(t, expectedAllDiskMetrics, numDiskMetrics)
 
 	tags1 := map[string]string{
-		"path":   "/",
+		"path":   string(os.PathSeparator),
 		"fstype": "ext4",
 		"device": "sda",
 		"mode":   "ro",
 	}
 	tags2 := map[string]string{
-		"path":   "/home",
+		"path":   fmt.Sprintf("%chome", os.PathSeparator),
 		"fstype": "ext4",
 		"device": "sdb",
 		"mode":   "rw",
```

Comment:
The locations will not exist on Windows anyway, so I guess we need a list of locations per OS. Same for all the other locations below.

Reply:
For this test these `path`s are only used as tags. It is not important if these locations exist or not. In `AssertContainsTaggedFields` equality of these tags is checked - it will fail if wrong `PathSeparator` is used in path.